app/views/claims/support/claims/index.html.erb

```diff
@@ -1,5 +1,38 @@
 <%= render "claims/support/primary_navigation", current: :claims %>
+<%= content_for :page_title, t(".heading") %>
 
 <div class="govuk-width-container">
-  <h1 class="govuk-heading-l"><%= t(".heading") %></h1>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l"><%= t(".heading") %></h1>
+
+    <% if @claims.any? %>
+      <table class="govuk-table">
+        <thead class="govuk-table__head">
+          <tr class="govuk-table__row">
+            <th scope="col" class="govuk-table__header"><%= t(".id") %></th>
+            <th scope="col" class="govuk-table__header"><%= t(".state") %></th>
```

Comment:
To be consistent with other BAT apps, let's also use "Status" instead of "State".

As model attributes, I think we can also use `Claim.human_attribute_name(:id)` and add the attribute localisation values to `activerecord.yml`.

![CleanShot 2024-02-06 at 12 29 50](https://github.com/DFE-Digital/itt-mentor-services/assets/4231530/2c0e22c6-fdbc-47f3-91ec-3734b30dab8d)

Reply:
Fixed https://github.com/DFE-Digital/itt-mentor-services/pull/220/commits/e67a5723e9988f8d55656d016a69c555972213c4